x-pack/plugins/enterprise_search/server/routes/app_search/credentials.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { IRouteDependencies } from '../../plugin';
+import { createAppSearchRequestHandler } from './app_search_request_handler';
+
+export function registerCredentialsRoutes({ router, config, log }: IRouteDependencies) {
+  router.get(
+    {
+      path: '/api/app_search/credentials/collection',
```

Comment:
Let's simplify our API routes at all if possible - I'm not a huge fan of how many of our internal API routes end in `/collection` or `/details` (I think originally due to hash routing conflicts?), and now they definitely don't have to be

```suggestion
      path: '/api/app_search/credentials',
```